### PR TITLE
feat: nearby beacon list according to start time

### LINKED ISSUE #98: Change preference of shown nearby beacons
**Is your feature request related to a problem? Please describe.**
List of fetched beacons in nearby beacons is in the order of Upcoming -> Active.

**Describe the solution you'd like**
The Active beacons should be shown first

**Are you working on this? (Yes/No)**
Yes

### PR DESCRIPTION
**Pull request by ItsAdityaKSingh**
Fixes #98 

Describe the changes you have made in this PR - Added preference to show active beacons earlier than upcoming ones in nearby beacons. Also added some commas.

Screenshots of the changes (If any) - 

Note: Please check Allow edits from maintainers. if you would like us to assist in the PR.

### CODE REVIEW

File: lib/services/database_mutation_functions.dart [hunk 1]
```diff
@@ -91,10 +91,16 @@ class DataBaseMutationFunctions {
 
   Future<String> signup({String name, String email, String password}) async {
     final QueryResult result = email != null
-        ? await clientNonAuth.mutate(MutationOptions(
-            document: gql(_authQuery.registerUser(name, email, password))))
+        ? await clientNonAuth.mutate(
+            MutationOptions(
+              document: gql(_authQuery.registerUser(name, email, password)),
+            ),
+          )
```
Comment (nb9960) [inline lib/services/database_mutation_functions.dart:98]: Avoid committing unnecessary changes. This issue requires just 1 line change

**Review by nb9960 — COMMENTED** [review]
Avoid committing unnecessary changes. This issue requires just 1 line change

**Comment by ItsAdityaKSingh** [comment]
> Avoid committing unnecessary changes. This issue requires just 1 line change

Apologies for the inconvenience. I have made the required changes.

**Review by nb9960 — APPROVED** [review]
